wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -581,20 +581,42 @@ class Meta:
 
 class PageChooserBlock(ChooserBlock):
 
-    def __init__(self, target_model='wagtailcore.Page', can_choose_root=False,
+    # TODO: rename target_model to page_type
+    def __init__(self, target_model=None, can_choose_root=False,
                  **kwargs):
-        self._target_model = target_model
+        if target_model:
+            # Convert single string/model into a list
+            if not isinstance(target_model, (list, tuple)):
+                target_model = [target_model]
+        else:
+            target_model = []
+
+        self._target_models = target_model
         self.can_choose_root = can_choose_root
         super(PageChooserBlock, self).__init__(**kwargs)
 
     @cached_property
     def target_model(self):
```

Comment:
Ah, now I see that the base `ChooserBlock` relies on the `target_model` property to do the conversion between ID and instance. (And, in fact, your chosen solution of returning `wagtailcore.Page` in the case of multiple page types is the only thing that can possibly work for [`ChooserBlock.bulk_to_python`](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/blocks/field_block.py#L538).) OK, let's keep it this way :-)